legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-snowflake/src/main/java/org/finos/legend/engine/persistence/components/relational/snowflake/logicalplan/datasets/SnowflakeStagedFilesDatasetPropertiesAbstract.java

```diff
@@ -36,4 +37,18 @@ public interface SnowflakeStagedFilesDatasetPropertiesAbstract extends StagedFil
     Optional<FileFormat> fileFormat();
 
     Map<String, Object> copyOptions();
+
+    @Value.Derived
+    default boolean dryRunSupported()
+    {
+        // Only supported for CSV
+        boolean dryRunSupported = false;
+        if (fileFormat().isPresent() && fileFormat().get() instanceof StandardFileFormatAbstract)
```

Comment:
Let's use StandardFileFormat instead of StandardFileFormatAbstract

Reply:
Sure, changed